test/runtests.jl

```diff
@@ -67,6 +67,32 @@ solution_ref_dense_from_data  = madnlp(lq_dense_from_data, max_iter=100)
 @test solution_ref_sparse_from_data.solution[(ns * (N + 1) + 1):(ns * (N + 1) + nu*N)] ≈ solution_ref_dense_from_data.solution atol =  1e-6
 
 
+# Test mul! operators and LQJacobianOperator
+
+Random.seed!(10)
+x = rand(nu * N)
+y = rand(size(lq_dense.data.A, 1))
+x_trunc = copy(x)
+y_trunc = copy(y)
+
+lq_dense_trunc = DenseLQDynamicModel(dnlp;truncated=true)
+
+Jac = lq_dense_trunc.data.A
```

Comment:
I would suggest implementing a proper getter to access the Jacobian inside a `DenseLQDynamicModel`